Source/relay/TourGuide/Quests/Level 6.ash

```diff
@@ -72,12 +73,12 @@ float QLevel6TurnsToCompleteArea(location place)
         turns_remaining = 10000.0; //how do you refer to infinity in this language?
 
     int max_turns_remaining = ncs_remaining * 5;
-    if (place == $location[the dark neck of the woods])
```

Comment:
we should probably actually delete the code block from base_quest_state re: "turns on last nc" if we're just gonna use the lastFriarsXNC prefs going forward. also might want to push required version # but that's a whole other can of worms.

Reply:
You're definitely right on both accounts. I think I put this together before we had the conversation about maintaining backwards compatibility and bumping the required version, so it wasn't on my mind but it should be done